controllers/goharbor/core/deployments.go

```diff
@@ -119,9 +119,16 @@ func (r *Reconciler) GetDeployment(ctx context.Context, core *goharborv1alpha2.C
 		scheme = "https"
 	}
 
+	host := r.NormalizeName(ctx, core.GetName())
+	if core.Spec.Components.TLS.Enabled() {
+		host += ":443"
+	} else {
+		host += ":80"
```

Comment:
why do we need to set such default ports?

the protocol scheme can not resolve the problem?

Reply:
because weiwei concat the port forcely with this PR
https://github.com/aquasecurity/harbor-scanner-trivy/issues/108
![image](https://user-images.githubusercontent.com/188115/100719000-ffb5ad80-33f6-11eb-9b8c-865f9149b852.png)